AdobeStockImage/etc/Model/ResourceModel/Asset/Collection.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImage\Model\ResourceModel\Asset;
+
+use Magento\AdobeStockImage\Model\Asset as Model;
+use Magento\AdobeStockImage\Model\ResourceModel\Asset as ResourceModel;
+use Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection;
+
+class Collection extends AbstractCollection
```

Comment:
Additional tables containing asset's attributes should be handled by asset's resource classes

Reply:
Hi @sivaschenko, I didn't understand what you said. Can you provide me an example?

Reply:
Sorry @igor-imaginemage , I mean that a part of data for an asset instance (like keywords, category, creator) are stored in separate tables, and the resource collection should include the logic to load this data as well. However, I believe it can be added in a separate pull request.

Reply:
Okay @sivaschenko, got it. So should I keep the file as we have right now or should I implement the logic to load the collection?